I/Ipopt/build_tarballs.jl

```diff
@@ -47,8 +47,8 @@ products = [
 # Dependencies that must be installed before this package can be built
 dependencies = [
     Dependency("ASL_jll"),
-    Dependency("OpenBLAS32_jll"),  # Ipopt uses 32-bit ints
-    Dependency("MUMPS_seq_jll"),
+    Dependency("OpenBLAS32_jll", v"0.3.9"),  # Ipopt uses 32-bit ints
```

Comment:
What 0.3.9 and not 0.3.10 or 0.3.12?

Reply:
No Julia version uses OpenBLAS 0.3.10, Julia v1.6 will use 0.3.12.  Julia v1.5 uses OpenBLAS 0.3.9.  If we're lucky and they're all ABI compatible the version doesn't really matter, but this it's better to use the oldest version possible (at least the one used in the current stable Julia version)